.gitlab/e2e_test_junit_upload.yml

```diff
@@ -39,6 +39,8 @@ e2e_test_junit_upload:
     - new-e2e-agent-platform-install-script-centos-fips-a7-x86_64
     - new-e2e-agent-platform-install-script-centos-fips-iot-agent-a7-x86_64
     - new-e2e-agent-platform-install-script-centos-fips-dogstatsd-a7-x86_64
+    - new-e2e-agent-platform-step-by-step-debian-a7-x64
+    - new-e2e-agent-platform-step-by-step-debian-a7-arm64
```

Comment:
Is there a reason to upload only the debian test results?

Reply:
missed that, adding it 👍